app/models/concerns/purgable.rb

```diff
@@ -0,0 +1,78 @@
+# Copied from the deprecated fastly-rails gem
+# https://github.com/fastly/fastly-rails/blob/master/lib/fastly-rails/active_record/surrogate_key.rb
+#
+# This concern handles purge and purge_all calls to purge the edge cache (Fastly)
+module Purgable
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def purge_all
+      return unless Rails.env.production?
```

Comment:
Oof, I want a QA person so we can get rid of this.

Reply:
Agreed 😬

Reply:
Also, documenting this here for future reference, we do this in the `CacheBuster` too:
https://github.com/thepracticaldev/dev.to/blob/9ab1979ef0c9c5ea0d400e855d13daae31869d60/app/labor/cache_buster.rb#L10

Reply:
Wouldn't that happen for everything? If I set up production credentials for other services I can do all sorts of things. :D

Reply:
BTW the main problem of all these guards, is that they can't be easily tested :(

Reply:
Yes and no, right? Some 3rd parties have dedicated environments (i.e. the URL is entirely different). I'll remove them, it'll just throw an error because we can't authenticate anyways in development.